src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -324,18 +314,9 @@ public boolean hasTutorPrivileges() {
         return privileges.hasTutorPrivileges();
     }
 
-    /**
-     * pre-condition: instructorPrivilegesAsText and privileges should be non-null
-     * @param instructor
-     * @return
-     */
```

Comment:
I don't think you should remove the header comment entirely. If it's no longer relevant, update it.